packages/go_router_builder/example/lib/shell_route_example.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Can you add a simple test for this example? Just make sure things will compile and the shellRoute's widget is built

Reply:
Added test in [fb449ff](https://github.com/flutter/packages/pull/3439/commits/fb449ff275a80230fc6656298eca347e744f82e6)